AmplifyPlugins/Core/AWSPluginsCore/Model/Decorator/ModelIdDecorator.swift

```diff
@@ -61,3 +63,49 @@ public struct ModelIdDecorator: ModelBasedGraphQLDocumentDecorator {
         return document.copy(inputs: inputs)
     }
 }
+
+extension Model {
+
+    func graphQLInputForPrimaryKey(modelFieldName: ModelFieldName) -> String? {
+
```

Comment:
let's move this over to `Model+GraphQL` since the other similar method also lives there and then we're not duplicating `getFieldValue` in two places